frontend/src/components/Buttons/__tests__/MarkPlayedButton.spec.ts

```diff
@@ -1,4 +1,3 @@
-import { BaseItemDto } from '@jellyfin/sdk/lib/generated-client';
```

Comment:
Ignore this whole file for now and skip it from this commit please. We need to rewrite tests completely for vitest, they're here so they could be easily traced without git bisecting when the test suite rewrite is done (after ``vite`` branch gets merged to master).

Same applies to all the spec files, so if you had another PR in mind for all of those, ignore them.
PRs refactoring the testing suite to use vitets are welcome of course :).